clustertask/github-update-cd-repo/templates/clustertask.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
+metadata:
+  name: {{ include "stakater-github-update-cd-repo.name" . }}
+spec:
+  workspaces:
+    - name: source
+  params:
```

Comment:
we have 3 different naming conventions being followed here for the param names; can we fix it